focus-ios/Blockzilla/Settings/Cells/SettingsTableViewAccessoryCell.swift

```diff
@@ -4,7 +4,7 @@
 
 import UIKit
 
-class SettingsTableViewAccessoryCell: SettingsTableViewCell {
```

Comment:
I'm getting this error when building.
Inheritance from a final class 'SettingsTableViewCell'